buggy-queries.txt

```diff
@@ -0,0 +1,2 @@
+what is the average salary of employees where name is not vinh
+how many jobs where name is not vinh
```

Comment:
Wait so what's the deal with the first one? It null pointers on QueryBuilder:106 - seems like the join recipe wherePrefix() is null pointering